src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -147,7 +147,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
     with self.argument_context('acr run') as c:
         c.argument('registry_name', options_list=['--registry', '-r'])
         c.positional('source_location', help="The local source code directory path (e.g., './src') or the URL to a git repository (e.g., 'https://github.com/Azure-Samples/acr-build-helloworld-node.git') or a remote tarball (e.g., 'http://server/context.tar.gz').", completer=FilesCompleter())
-        c.argument('file', help="The task template/definition file path relative to the source context.")
+        c.argument('file', options_list=['--file', '-f'], help="The task template/definition file path relative to the source context.")
```

Comment:
@ehotinger Do we want to keep `-f` option here since it cannot be a `Dockerfile`?

Reply:
yes, it'll allow you to interchange `az acr build` and `az acr run`